tests/conftest.py

```diff
@@ -188,3 +220,71 @@ def mock_RegisteredEntrypointMetadata(
 ) -> RegisteredEntrypointMetadata:
     """Return a RegisteredEntrypointMetadata object populated with test data."""
     return RegisteredEntrypointMetadata(**entrypoint_metadata_json)
+
+
+@pytest.fixture
```

Comment:
I haven't seen the faker library before but now I'm intrigued, this looks very handy

Reply:
reminds me of another testing library I've never used but seems like a fun rabbit hole: https://hypothesis.readthedocs.io/en/latest/manifesto.html